index.html

```diff
@@ -3821,37 +3822,37 @@ <h2>DID Resolution</h2>
 method-specific internal function to perform the actual <a>DID resolution</a>
 process. <a>DID resolver</a> implementations might implement and expose
 additional functions with different signatures in addition to the
-<code>resolve</code> function specified here.
+<code>resolve</code> and <code>resolveRepresentation</code> functions specified here.
```

Comment:
non blocking editorial comment: Why mention that they can be implemented differently here? This seems to encourage extensions that could lead to divergence at the API level and ultimately cause interop concerns. I think this is best removed.

Reply:
It's actually saying the opposite of that: you can't implement `resolve` and `resolveRepresentation` with different signatures at all, you always have to use these signatures. However you can also add additional functions with different signatures and expose those to your callers -- and nothing we say in the spec would stop that, so it's best to clearly admit it here.